practice/react-practice-one/src/components/Text/Text.jsx

```diff
@@ -23,6 +30,7 @@ const Text = ({
 Text.propTypes = {
   variant: PropTypes.oneOf(Object.values(TEXT_VARIANTS)),
   size: PropTypes.oneOf(Object.values(COMPONENT_SIZES)),
+  type: PropTypes.oneOf(Object.values(COMPONENT_TEXTS)),
```

Comment:
Please rename the constants; they are confusing me

Reply:
I changed it to `LABELS`. I don't know if it's okay ?